<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SYSTEM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBookAdmin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>630</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add book (NEW) admin only </string>
    </property>
   </widget>
   <widget class="QLabel" name="bookPreview">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>221</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>306</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>All Books</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_books">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>ewqewq</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ewqeqw</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ewqewq2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>258</width>
      <height>232</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="authLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="bookContentPlain"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>122</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="createAccountBtn">
       <property name="text">
        <string>Create Account</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>160000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Profile Settings | Status :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginBtn">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signOutBtn">
       <property name="text">
        <string>Sign out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="loginLb">
       <property name="text">
        <string>Not logged in</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget_books</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>titleLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
